core/genesis.go

```diff
@@ -313,16 +313,21 @@ func SetupGenesisBlockWithOverride(db ethdb.Database, triedb *trie.Database, gen
 			// }
 
 			// [Kroma: START]
-			zero := uint64(0)
 			if config.IsKroma() {
+				// Load the chain-config for the given chain id, and overrides it if it exists.
+				if overrides != nil && config.ChainID != nil && config.ChainID.IsUint64() {
```

Comment:
I think we don't need to check `overrides` here.
```suggestion
				if config.ChainID != nil && config.ChainID.IsUint64() {
```